homeassistant/components/dnsip/sensor.py

```diff
@@ -89,7 +89,7 @@ async def async_update(self) -> None:
             response = None
 
         if response:
-            self._attr_native_value = response[0].host
+            self._attr_native_value = ",".join(sorted([i.host for i in response]))
```

Comment:
States are limited to 255 characters. This will cause a problem if there are many a records

Reply:
Hi. A IP is max 15 characters (if all ip-tupples are 3 characters) 100.100.100.100. That leaves us with 17 A-records. That is waaaay higher that I've ever seen. Max I've seen in the wild is around 8 (Heroku LB's) So I do not think that will be a problem.

Reply:
Should probably put in a limit anyhow